packages/http-client-python/generator/pygen/codegen/templates/model_dpg.py.jinja2

```diff
@@ -70,7 +70,7 @@
     {% endif %}
     {% set initialize_properties = serializer.initialize_properties(model) %}
     {% if serializer.need_init(model) or initialize_properties %}
-    def __init__(self, *args: Any, **kwargs: Any) -> None:{{ '# pylint: disable=useless-super-delegation' if not initialize_properties else '' }}
+    def __init__(self, *args: Any, **kwargs: Any) -> None:{{ serializer.pylint_disable(model) }}
```

Comment:
It is introduced in https://github.com/microsoft/typespec/pull/4571/files which to fix CI errors of autorest.python repo. Please hold the PR until I fix CI issue https://github.com/microsoft/typespec/issues/4570.

Reply:
Update: after https://github.com/microsoft/typespec/pull/4648 merged, CI will indeed report error when lint issues exist in generated SDK code (e.g. [here](https://github.com/microsoft/typespec/pull/4650/checks?check_run_id=31275862070))

Reply:
i had removed it because with `next-pylint`, we're getting `useless-suppression` errors there

Reply:
https://github.com/microsoft/typespec/pull/4648 is merged and CI can check lint issues now.